src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -203,6 +203,11 @@ public string HubName
         /// </summary>
         public bool UseAppLease { get; set; } = true;
 
+        /// <summary>
+        /// If <c>true</c>, the inputs of activity and sub-orchestrator functions will be saved to the orchestration history, making them visible to debugging tools. The default value is <c>false</c>.
```

Comment:
We should probably keep it general to cover all functions like entities. We could say something like "functions called by the orchestrator" instead of activity and sub-orchestrator functions.

Reply:
Thanks!! I will make it clear.